client/utils/auth.py

```diff
@@ -131,11 +161,23 @@ def authenticate(assignment, force=False):
         except Exception as _:
             print('Performing authentication')
 
-    network.check_ssl()
+    try:
+        access_token, expires_in, refresh_token = get_code(assignment)
+        update_storage(access_token, expires_in, refresh_token)
+        return access_token
+    except OAuthException as e:
+        print("Authentication error: {}".format(e.error.replace('_', ' ')))
```

Comment:
The error doesn't really stand out. Do you think it should be separated/call more attention?

````
Paste your code here: badgrant
Authentication error: invalid grant
Not authenticated. Cannot send backup to server
Checking for software updates...
OK is up to date
````

If you want to draw attention to it - `utils.format` has a context manager `block` that makes it easy to do something like
```
-------------------- 
Authentication Error: ....
--------------------
```

(Also - should this exception be logged? I'm leaning towards no since there's no new information that's not already being printed or logged earlier when the OAuthException is thrown)